website/docs/compute/customizing-functions/fully-qualified-function-names.md

```diff
@@ -1,24 +1,30 @@
+---
+pagination_next: null
+---
+
 # Fully-qualified function names
 
 Each function uploaded by your users has a unique name called a fully-qualified function name, or FQFN. The string representation of an FQFN is as follows:
 
 ```bash
-com.awesomeco.nawronuq98hqwekj198fkljbeco#default::record-event@v1.0.0
-|____________|___________________________|________|____________|______|
+com.example.nawronuq98hqwekj198fkljbeco#default::record-event@v1.0.0
+|_____________|___________________________|________|____________|______|
  Environment          User ID             Namespace  Function   Version
 ```
 
 Here's a breakdown:
 
-* **Environment**: your company's reverse domain; `com.awesomeco`
+* **Environment**: your company's reverse domain; `com.example`
 * **User ID**: The unique value that your application uses to identify your users within your system; `nawronuq98hqwekj198fkljbeco`
-* **Namespace**: The namespace this function belongs to; allows for separated groups of functions based on product needs \(see [Namespaces](namespaces.md) \); `default`
+* **Namespace**: The namespace this function belongs to; allows for separated groups of functions based on product needs (see [Namespaces](docs/compute/customizing-functions/namespaces.md) \); `default`
```

Comment:
Is escaping the closing paren necessary?

Reply:
Nope! Will fix.